api/src/pcapi/routes/serialization/finance_serialize.py

```diff
@@ -67,7 +67,7 @@ class Config:
     def from_orm(cls, invoice: finance_models.Invoice):
         invoice.businessUnitName = invoice.businessUnit.name
         res = super().from_orm(invoice)
-        res.amount /= 100
+        res.amount = abs(res.amount / 100)
```

Comment:
On a une fonction de conversion centimes->euros qu'il est préférable d'utiliser. De plus, mieux vaudrait prendre l'opposé plutôt que la valeur absolue, pour justement avoir un montant négatif côté frontend si jamais on a une facture avec un montant positif (indiquant une somme d'argent à payer **par** l'acteur culturel, même si c'est impossible actuellement, et très improbable dans le futur).


```suggestion
        res.amount = -finance_utils.to_euros(res.amount)
```

(avec `import core.finance.utils as finance_utils`)